kolibri/plugins/coach/assets/src/views/reports/ReportsLessonExerciseLearnerListPage.vue

```diff
@@ -271,8 +271,8 @@
           .reduce((entries, groupEntries) => entries.concat(groupEntries), []);
 
         if (this.ungroupedEntries.length) {
-          data.concat(
-            this.ungroupedEntries.map(entry => {
+          data.push(
```

Comment:
I don't recall having seen using `arr.push(...someArray)` in place of `arr.concat(someArray)` before but _I like it_!

Reply:
Did you make this change as a style change or do you know if this improves performance or something?

Reply:
Playing with it a little bit I realize that the `push` version mutates the object itself whereas `concat` returns a whole new array so was the previous code an issue because of that?

Reply:
Yeees!! The error why ungrouped learners didnt appear in the report was because the `concat` method doesnt work in-place, and we werent reassigning the result, but as `data` was a constant, and we couldnt reassign I preferred to use the `push(...array)` instead of defining the constant as variable.